src/scripts/Update.ts

```diff
@@ -716,6 +716,12 @@ class Update implements Saveable {
             // Remove the Elite_Nihilego Gym, now a temporary battle instead of a gym
             saveData.statistics.gymsDefeated.splice(84, 1);
             saveData.badgeCase.splice(84, 1);
+            // Change Ultra Megalopolis to a Temporary Battle
+            saveData.statistics.temporaryBattleDefeated = new Array<number>();
+            saveData.statistics.temporaryBattleDefeated[1] = saveData.statistics.gymsDefeated[89];
```

Comment:
Because Nihilego have been removed earlier in the update function, this gym now has index 88 and not index 89